Update migration docs to work for Hosted (#2369)
astronomer/docs: learn/marquez.md

## Conclusion

> Congratulations! You are now able to run Marquez and Airflow locally and trace data through your DAGs by viewing their lineage. A great next step is to try other Airflow operators that generate lineage metadata, or, if you are an Astronomer customer, check out [lineage in Astro](https://docs.astronomer.io/astro/data-lineage).

Reviewer: ```suggestion
Congratulations! You can now run Marquez and Airflow locally and trace data through your DAGs by viewing their lineage. As a great next step, try other Airflow operators that generate lineage metadata. Or, if you are an Astronomer customer, check out [lineage in Astro](https://docs.astronomer.io/astro/data-lineage).
```
The last sentence felt like a run-on, I could be wrong though!
